lib/sms.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+
+class SMS
+
+  def send_sms
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for send_sms is too high. [15.65/15] (http://c2.com/cgi/wiki?AbcMetric)<br>Metrics/MethodLength: Method has too many lines. [12/10] (https://github.com/bbatsov/ruby-style-guide#short-methods)